spotinst_sdk2/clients/elastigroup/__init__.py

```diff
@@ -1767,59 +1753,57 @@ def get_cost_per_elastigroup(self, group_id, to_date=None, from_date=None):
 
         return formatted_response["response"]["items"]
 
-    def get_elastigroup_activity(self, group_id, start_date):
+    def get_elastigroup_activity(self, group_id: str, start_date: str, end_date: str):
         """
-        Get elastigroup activity
+        Get all activity events for the Elastigroup
 
         # Arguments
         group_id (String): Elastigroup ID
         start_date (String): Date when to start checking
+        end_date (String): Get items on or before this date (ISO 8601)
 
         # Returns
         (Object) : Elastigroup API response 
         """
         query_params = self.build_query_params_with_input(
-            {"fromDate": start_date})
+            {"fromDate": start_date, "toDate": end_date})
 
         content = self.send_get(
-            url=self.__base_elastigroup_url +
-            "/" +
-            str(group_id) +
-            "/events",
+            url=self.__base_elastigroup_url + "/" + str(group_id) + "/events",
             query_params=query_params,
             entity_name='active events')
 
         formatted_response = self.convert_json(
             content, self.camel_to_underscore)
         return formatted_response["response"]["items"]
 
-    def import_gke(self, location, gke_id, gke):
+    def import_gke(self, location: str, gke_id: str, node_pool_name: str, gke: gcp_gke.GKE):
```

Comment:
rename gke last parameter to import_config or eg_config

Reply:
Done